v0.7/mlperf.conf

```diff
@@ -45,8 +46,11 @@ ssd-resnet34.Server.target_latency = 100
 gnmt.Server.min_query_count = 90112
 gnmt.Server.target_latency = 250
 gnmt.Server.target_latency_percentile = 97
+rnnt.Server.min_query_count = 90112
+rnnt.Server.target_latency = 13000
+rnnt.Server.target_latency_percentile = 97
```

Comment:
Should this be 99 or 97? @christ1ne @guschmue

Reply:
I believe 99 is the default. If there is some special circumstance or some experimental data, we can change it.

Reply:
There's a private discussion I had with David about this:

-------

Specifying a maximum input length sounds good to me. I plotted a histogram of the librispeech dev-clean dataset here:

![image](https://user-images.githubusercontent.com/4767568/83457614-309c3200-a416-11ea-99d6-1c339616373c.png)

Librispeech has longer utterances than, for example, google voice search, because each utterance is typically a sentence in a book, which have been spoken by professional book readers, who usually go slow for comprehensibility. The above graph demonstrates this. I plotted some percentiles as well. So 13 seconds is the 90% percentile.

If we choose to filter out all data above the 90% percentile, we can probably use a latency constraint of 13 seconds (the 90% percentile) as a start. On today's server class computers, 1x real-time factor is a very generous constraint for a DNN-based speech-to-text system. So this will be a very "easy" constraint to reach, especially since most all samples will be less than 13 seconds in length. Keep that in mind. For edge, this may be hard to reach, though.

It is probably worthwhile to point out at this point that these durations are much, much longer than any other benchmark's durations, as far as I can tell.

At 13 seconds per sample and 97 percentile latency, that's 90,112 inferences * 13 seconds / inference = 325 hours, which is not reasonable. That is probably the biggest problem to figure out right now. Of course, most inferences will take less than that 13 second constraint. However, it's probably still unreasonable for most submitters to deal with successfully.

-------

Basically, because the sequence length of ASR inputs is so long, I would expect that even 97th percentile will be too slow.

Also note that this https://github.com/mlperf/inference_policies/blob/master/inference_rules.adoc#3-scenarios suggests 97 percentile for non-image models.

Reply:
@galv can you send me one slide which summarizes all the decisions needed on the latency target and your proposal to be used in the meeting tomorrow? Thanks! @nvpohanh